dll/cpl/sysdm/environment.c

```diff
@@ -11,6 +11,7 @@
 #include <commctrl.h>
 #include <commdlg.h>
 #include <string.h>
+#include <lmcons.h>
 
```

Comment:
```suggestion
#include <strsafe.h>

```
Used for ``StringCchPrintfW``

Reply:
Ok. Is safer to use StringCchPrintfW but we need to include lmcons.h to define UNLEN.

Reply:
Where do you need UNLEN?

Reply:
WCHAR szUserName[UNLEN + 1] = {0}; ????

Reply:
It may have another name like MAX_STR_LENGTH !

Reply:
But then, why would you want to use static buffers 65535 bytes long?

Reply:
> Ok. Is safer to use StringCchPrintfW but we need to include lmcons.h to define UNLEN.

I did not excluded it. I've just added strsafe.

Reply:
https://github.com/reactos/reactos/pull/4996#discussion_r1092627134

Good question!

Reply:
> [#4996 (comment)](https://github.com/reactos/reactos/pull/4996#discussion_r1092627134)
> 
> Good question!

I did not understood the question... Can you reformulate ?
AFAIK we dont use a 65535 bytes long static buffer ?